University-advisor-web/Tools/Logger.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Elasticsearch.Net;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using University_advisor_web.Interfaces;
+using University_advisor_web.Models;
+
+namespace University_advisor_web.Tools
+{
+    public class Logger : ILogger
+    {
+        public void Log(string message, string level = "INFO")
+        {
+            var messageLog = new LogBodyModel
+            {
+                Timestamp = DateTime.Now.ToString("MM/dd/yyyy HH:mm:ss"),
```

Comment:
Maybe we should use our local date format instead of american one?

Reply:
You mean day-month-year?

Reply:
Yes. Or year month day